docs/privacy/data-handling.md

```diff
@@ -0,0 +1,111 @@
+
+Production data must only be shared with approved members of the engagement team and must not be processed/transferred outside of the customer controlled environment.
+
+Customers should provide CSE with a copy of the requested data in a location managed by the customer.
+The customer should consider turning any logging capabilities on so they can clearly identify who has access and what they do with that access.
+CSE should notify the customer when they are done with the data and suggest the customer destroy copies of the data if they are no longer needed.
+
+Our guiding principles when handling data in an engagement include:
+
+- Only provide access to production data when required and explicitly state 5Ws behind the data access.
+- Limit access to the data set required to meet the engagement objectives.
+- Explicitly state the intended purpose of data that can be used for engagement.
+- Only share production data with the approved members of the engagement team.
+- The entire team should work together to ensure that there are no dead copies of data. When the data is no longer needed,
+the team should promptly work to clean up engagement copies of data.
+- Do not send any production data outside the customer-controlled environment.
+
+Questions to consider when working with data:
+
+- What data do we need?
+- What is the legal basis for processing this data?
+- If we are the processor based on contract obligation what is our responsibility listed in the contract?
+- Does the contract need to be amended?
+- How can we contain data proliferation?
+- What security controls are in place to protect this data?
+- What is the data breech protocol?
+- How does this data benefit the data subject?
+- What is the lifespan of this data?
+- Do we need to keep this data linked to a data subject?
+- Can we turn this data into Not in a Position to Identify (NPI) data to be used later on?
+- How is the system architected so data subject rights can be fulfilled? (ex manually, automated)
+- If personal data is involved have engaged privacy and legal teams for this project?
+It is important to only pull in data that is needed for the problem at hand,
+when this is put in practice we find that we only maintain data that is adequate,
+relevant and limited to what is necessary in relation to the purposes for which they are processed.
+This is particularly important for personal data. Once you have personal data there are many rules and regulations that apply,
+some examples of these might be HIPPA, GDPR, CCPA.
+The customer should be aware of and surface any applicable regulations that apply to their data.
+Furthermore the [seven principles of privacy by design](http://dataprotection.industries/wp-content/uploads/2017/10/privacy-by-design.pdf)
+should be reviewed and considered when handling any type of sensitive data.
+Additional information:
+
+- [Microsoft Trust Center](https://www.microsoft.com/en-us/trust-center/privacy)
```

Comment:
Are these last bullet points are a subset of the last question? If so, maybe adding some tabs would improve readability here?

Reply:
Agreed. Done